pyproject.toml

```diff
@@ -646,4 +644,6 @@ module = [
     'twisted.python.test.test_htmlizer',
     'twisted.application.test.test_service',
     'twisted.python.randbytes',
+    'twisted.python.zippath',
+    'twisted.python.filepath',
```

Comment:
hmm, this feels like a place we should be strictly removing stuff from a list, rather than adding to one. do you know if there's a reason it was set up this way?

Reply:
Yeah I think we should make disallow untyped defs the default